### DISCUSSION TITLE
About qMultiObjectiveMaxValueEntropy

### DISCUSSION THREAD

**Discussion by LiuYang-1997** [meta-pytorch/botorch#1066] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/58632163/150540653-8a8f9578-07c4-4a14-8db7-9f3801364ef5.png)
Help me!
I've been confused all day.
Which module is responsible for creating it?

**Comment by sdaulton** [comment, marked as answer]
Apologies. I had intended to put out a tutorial for this, but it ended up on the back burner. There currently isn't a utility for this in botorch. The MESMO authors propose to use NSGA-II to optimize approximate GP sample paths (using RFFs), which I would recommend but this would require an external package such as pymoo. Alternatively, if the problem is low-dimensional, you may be able to get away with densely random sampling designs from the entire search space to optimize RFF samples (you would still want to use RFFs to avoid sampling from a huge joint posterior over densely random sampled designs). I will aim to put up a tutorial for this in early February.

**Reply by LiuYang-1997** [reply]
Thank you for your hard work.

**Reply by LiuYang-1997** [reply]
Sorry to bother you, but I would like to know when this tutorial will be available
